src/ellipsed.js

```diff
@@ -9,25 +9,25 @@
  */
 
 function tokensReducer(acc, token) {
-  const { el, elStyle, elHeight, rowsLimit, rowsWrapped } = acc;
+  const { el, elStyle, elHeight, rowsLimit, rowsWrapped, options } = acc;
   if (rowsWrapped === rowsLimit + 1) {
     return { ...acc };
   }
   const textBeforeWrap = el.textContent;
   let newRowsWrapped = rowsWrapped;
   let newHeight = elHeight;
   el.textContent = el.textContent.length
-    ? `${el.textContent} ${token}...`
-    : `${token}...`;
+    ? `${el.textContent} ${token}${options.replaceStr}`
+    : `${token}${options.replaceStr}`;
 
   if (parseFloat(elStyle.height) > parseFloat(elHeight)) {
     newRowsWrapped++;
     newHeight = elStyle.height;
 
     if (newRowsWrapped === rowsLimit + 1) {
       el.innerHTML = textBeforeWrap[textBeforeWrap.length - 1] === '.'
-        ? `${textBeforeWrap}..`
-        : `${textBeforeWrap}...`;
+        ? `${textBeforeWrap}${options.replaceStr}`
```

Comment:
Good point, the code checks to see if the last character is a `.` and if so only adds two dots, instead of the default three?

I'll revert the change.

Reply:
Sorry, I made a mistake writing the line of code: what we were talking was in line 29, and if I'm not wrong, then you reverted the wrong change.